1.illumination_correction/README.md

```diff
@@ -0,0 +1,77 @@
+# Cell Profiler Illumination Correction Pipeline
+In this module, I present the 10 CellProfiler pipelines I created using various manual parameters to create the illumination function and correct the raw images. The chosen image for comparison is **D6_01_1_4_DAPI_001.tif** due to the large artifact present in the bottom right corner.
```

Comment:
```suggestion
In this module, I present the 10 CellProfiler pipelines I created using various manual parameters to create the illumination function and correct the raw images.
The chosen image for comparison is **D6_01_1_4_DAPI_001.tif** due to the large artifact present in the bottom right corner.
```

Reply:
Can you comment on the goal of the illumination function for this image? It's not clear to me that any amount of massaging this image will adjust for this artifact. Are we expecting to keep this cell? Are we expecting to keep the other cells?